gradio/routes.py

```diff
@@ -803,11 +803,11 @@ async def sse_stream(request: fastapi.Request):
                         message=str(e),
                     )
                     response = process_msg(message)
-                    if response is not None:
```

Comment:
Ah so we already had the deletion logic in place, but it was never called?

Reply:
Yea we yielded the response before the clean up logic happened so we never reached that point in the code

Reply:
Nice catch